nipype/interfaces/ants/registration.py

```diff
@@ -395,6 +397,8 @@ class RegistrationOutputSpec(TraitedSpec):
     warped_image = File(desc="Outputs warped image")
     inverse_warped_image = File(desc="Outputs the inverse of the warped image")
     save_state = File(desc="The saved registration state to be restored")
+    metric_value = traits.Float(desc='the final value of metric')
+    elapsed_time = traits.Float(desc='the total elapsed time as reported by ANTs')
```

Comment:
Elapsed time is calculated by ANTs so it refers exclusively to the registration process. The duration of the runtime object adds the (little) overhead of nipype. In general, they will be equivalent, but this could be a good handle to catch problems.

Additionally, my plan would be to add the elapsed_time per registration stage at some point. Then, it will make sense to be consistent with ANTs calculations.